bin/fusioninventory-win32-service

```diff
@@ -6,206 +6,70 @@ use warnings;
 use lib './lib';
 use setup;
 
-use threads;
-use threads 'exit' => 'threads_only';
-
-use constant SERVICE_SLEEP_TIME  => 200; # in milliseconds
-use constant SERVICE_NAME        => "fusioninventory-agent";
-use constant SERVICE_DISPLAYNAME => "Fusioninventory Agent";
-
-use POSIX ":sys_wait_h";
 use File::Spec;
-use File::Basename;
 
-use English qw(-no_match_vars);
 use Getopt::Long;
 use Pod::Usage;
-use UNIVERSAL::require;
-
-use Win32;
-use Win32::Daemon;
 
-delete($ENV{PERL5LIB});
-delete($ENV{PERLLIB});
+use FusionInventory::Agent::Tools::Win32::Service;
 
 Getopt::Long::Configure( "no_ignorecase" );
 
-my $options = {};
+my %options = ();
 
```

Comment:
I just prefer to pass hash in place of hashref or dereferenced hash later when calling DeleteService() method at line 53. This is more readable imho to pass the hash. I really don't care to use reference or not, but I definitively don't like the not sexy %$hash notation.